repos/exporter/lib/atcoCheckerHandler.ts

```diff
@@ -0,0 +1,286 @@
+
+    // our inbound matching json stops, sort the stops and remove duplicates
+    knownInboundStops = [...new Set(knownInboundStops.sort())];
+
+    const outboundServicesRequiringAttention = [];
+    const inboundServicesRequiringAttention = [];
+
+    for (const key in outboundServiceAndSortedUniqueStopsFromFeed) {
+        const serviceId = key;
+        const stops = outboundServiceAndSortedUniqueStopsFromFeed[serviceId];
+
+        if (JSON.stringify(stops) !== JSON.stringify(knownOutboundStops)) {
+            console.log(`The stops on ${serviceId} require attention.`);
+            outboundServicesRequiringAttention.push(serviceId);
+        }
+    }
+
+    for (const key in inboundServiceAndSortedUniqueStopsFromFeed) {
+        const serviceId = key;
+        const stops = inboundServiceAndSortedUniqueStopsFromFeed[serviceId];
+
+        if (JSON.stringify(stops) !== JSON.stringify(knownInboundStops)) {
+            console.log(`The stops on ${serviceId} require attention.`);
+            inboundServicesRequiringAttention.push(serviceId);
+        }
+    }
+
+    return outboundServicesRequiringAttention.concat(inboundServicesRequiringAttention);
+};
+
+/**
+ * Builds and returns a hasmap (dictionary) where the service id is the key
+ * and the stops array is the value.
+ *
+ * @param {DirectionAndStops[]} directionsAndStopsArray
+ *
+ * @returns {{ [key: string]: string[] }} a hashmap (dictionary) with service id
+ * as the key and the stops array as the value
+ */
+const buildAHashmapSplitOnServiceId = (directionsAndStopsArray: DirectionAndStops[]): { [key: string]: string[] } => {
+    const dictionary: { [key: string]: string[] } = {};
+
+    directionsAndStopsArray.forEach((directionsAndStopsItem: DirectionAndStops) => {
+        const key = directionsAndStopsItem.serviceId;
+
+        const value: string[] | undefined = dictionary[key];
+
+        if (value !== undefined) {
+            // we already have that serviceId as key
+            value.push(directionsAndStopsItem.fromAtcoCode);
+            value.push(directionsAndStopsItem.toAtcoCode);
+        } else {
+            dictionary[key] = [directionsAndStopsItem.fromAtcoCode, directionsAndStopsItem.toAtcoCode];
+        }
+    });
+
+    return dictionary;
+};
+
+/**
+ * Sorts and remove duplicates stops from the stops array for each service id
+ *
+ * @param {serviceAndStopsDictionary} a hashmap (dictionary) with service id
+ *
+ * @returns {{ [key: string]: string[] }} a hashmap (dictionary) with service id
+ * as the key and unique and sorted stops array as the value
+ */
+const sortAndRemoveDuplicateStops = (serviceAndStopsDictionary: {
+    [key: string]: string[];
+}): { [key: string]: string[] } => {
+    for (const key in serviceAndStopsDictionary) {
+        const stopsArray = serviceAndStopsDictionary[key];
+
+        const sortedStopsArray = stopsArray.sort();
+
+        serviceAndStopsDictionary[key] = [...new Set(sortedStopsArray)];
```

Comment:
Same here with the sort ordering of the set. Could we fix this helper function and use this in the other places?